specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2018-02-14/keyvault.json

```diff
@@ -665,6 +665,187 @@
           "application/json"
         ]
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.KeyVault/vaults/{vaultName}/privateEndpointConnections/{privateEndpointConnectionName}": {
+      "get": {
+        "tags": [
+          "PrivateEndpointConnections"
+        ],
+        "operationId": "PrivateEndpointConnections_Get",
+        "description": "Gets the specified private endpoint connection associated with the key vault.",
+        "x-ms-examples": {
+          "KeyVaultGetPrivateEndpointConnection": {
+            "$ref": "./examples/getPrivateEndpointConnection.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/PrivateEndpointConnectionName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Private endpoint connection successfully returned.",
+            "schema": {
+              "$ref": "#/definitions/PrivateEndpointConnection"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "PrivateEndpointConnections"
+        ],
+        "operationId": "PrivateEndpointConnections_Put",
+        "description": "Updates the specified private endpoint connection associated with the key vault.",
+        "x-ms-examples": {
+          "KeyVaultPutPrivateEndpointConnection": {
+            "$ref": "./examples/putPrivateEndpointConnection.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/PrivateEndpointConnectionName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "properties",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PrivateEndpointConnection"
+            },
+            "description": "The intended state of private endpoint connection."
+          }
+        ],
+        "responses": {
```

Comment:
>responses [](start = 9, length = 9)

You should specify the `x-ms-long-running-operation` extension since this can be a long running operation ([[https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation)](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation))

Reply:
Same as before, copied from https://github.com/Azure/azure-rest-api-specs/blob/master/specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json. They also don't have `x-ms-long-running-operation`.

Why the storage code was accepted? Is this a new requirement?

Reply:
@majastrz 

Both SRP and AKV are following the guidance from https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creating-or-updating-resources:

> The API flow for PUT/PATCH should be to:
> 
> 1. Respond to the initial PUT request with a 201 Created or 200 OK (per normal guidance);
> 2. (...)
> 3. **Optional:** The response headers may include a Azure-AsyncOperation header pointing to an Operation resource

I was willing to add the `x-ms-long-running-operation`, however it requires us to return 202 instead of 200 and this is a breaking change. Please do not block the PR on this.